pcf/particle/aws/cloudfront/cloudfront.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pcf.core.aws_resource import AWSResource
+from pcf.core import State
+from pcf.util import pcf_util
+import time
+from datetime import datetime, timedelta
+
+
+class CloudFront(AWSResource):
+    """
+    Particle that maps to AWS Cloud Front
+    """
+
+    flavor = 'cloudfront'
+
+    START_PARAM_FILER = {
+        'CallerReference',
+        'Aliases',
+        'DefaultRootObject',
+        'Origins',
+        'OriginGroups',
+        'DefaultCacheBehavior',
+        'CacheBehaviors',
+        'CustomErrorResponses',
+        'Comment',
+        'Logging',
+        'PriceClass',
+        'Enabled',
+        'ViewerCertificate',
+        'Restrictions',
+        'WebACLId',
+        'HttpVersion',
+        'IsIPV6Enabled'
+    }
+
+    equivalent_states = {
+        State.running: 1,
+        State.stopped: 0,
+        State.terminated: 0,
+    }
+
+    UNIQUE_KEYS = ["aws_resource.Comments"]
+
+    def __init__(self, particle_definition):
+        """
+        Args:
+            particle_definition (definition): desired configuration of the particle
+        """
+        super(CloudFront, self).__init__(particle_definition, 'cloudfront')
+        self._id = ""
+        self._ifMatch = ""
```

Comment:
what are you trying to do here. If you are trying to default them, default to `None`